core/src/commonJvmMain/kotlin/com/ustadmobile/core/util/AssetUtil.kt

```diff
@@ -1,5 +1,6 @@
 package com.ustadmobile.core.util
 
 import java.io.InputStream
+import kotlin.reflect.KClass
 
-expect fun getAssetFromResource(path: String, context: Any): InputStream?
+expect fun getAssetFromResource(path: String, context: Any, clazz: KClass<*>): InputStream?
```

Comment:
Please add javadoc comment here specifying the parameters. E.g. here KClazz is what would be used to access the resource on JVM, and therefor the resource must be located inside the same module as the KClass